koji_containerbuild/cli.py

```diff
@@ -33,6 +34,18 @@
 # matches hub's buildContainer parameter channel
 DEFAULT_CHANNEL = 'container'
 
+def print_task_result(task_id, result, weburl):
+    try:
+        result = json.loads(result)
+        result["koji_builds"] = ["%s/buildinfo?buildID=%s" % (weburl, build_id)
+                                 for build_id in result.get("koji_builds", [])]
+        result = json.dumps(result, sort_keys=True, indent=4)
+    except:
+        print "Unable to convert result output to JSON."
```

Comment:
Remaining of file uses old-style print. Changing here requires changing for the whole file.